tools/topology/topology2/platform/intel/dmic-wov.conf

```diff
@@ -10,28 +10,22 @@ Object.Pipeline {
 			Object.Widget.host-copier.1 {
 				stream_name $DMIC1_PCM_CAPS
 				pcm_id 11
-				Object.Base.audio_format.1 {
-					in_rate			16000
-					in_bit_depth		32
-					in_valid_bit_depth	32
-					out_rate		16000
-					out_bit_depth		32
-					out_valid_bit_depth	32
-				}
-				Object.Base.audio_format.2 {
-					in_rate			16000
-					in_channels		4
-					in_bit_depth		32
-					in_valid_bit_depth	32
-					out_rate		16000
-					out_channels		4
-					out_bit_depth		32
-					out_valid_bit_depth	32
-					in_ch_cfg	$CHANNEL_CONFIG_3_POINT_1
-					in_ch_map	$CHANNEL_MAP_3_POINT_1
-					out_ch_cfg	$CHANNEL_CONFIG_3_POINT_1
-					out_ch_map	$CHANNEL_MAP_3_POINT_1
-				}
+				num_output_audio_formats 2
+				Object.Base.output_audio_format [
+					{
+						out_rate		16000
```

Comment:
@ranj063 This has no input_audio_format specified in this patch? How does this work out -- woulnd base/parent object set in_rate to 48000?

Tracking what comes from base and what is specified in locally, is not so easy to follow, especially with such a large PR...